homeassistant/components/efergy/sensor.py

```diff
@@ -50,12 +52,14 @@
         name="Energy Usage",
         device_class=DEVICE_CLASS_POWER,
         native_unit_of_measurement=POWER_WATT,
+        state_class=STATE_CLASS_MEASUREMENT,
```

Comment:
I don't think we should add new features to this integration. It's missing a service specific 3rd party library.

Reply:
https://github.com/home-assistant/core/commit/e078f4ce144e4168953143b5e7fc87cfccc83f81